package.json

```diff
@@ -10,8 +10,8 @@
     "lint": "eslint-godaddy src/*.js src/**/*.js test/*.js test/**/*.js",
     "postpack": "rm -f oclif.manifest.json npm-shrinkwrap.json",
     "prepack": "oclif-dev manifest && oclif-dev readme && npm shrinkwrap",
-    "pretest": "npm run lint",
-    "test": "nyc mocha --forbid-only \"test/**/*.test.js\"",
```

Comment:
👍

Reply:
I was trying to test out a single file and lint got in the way, ok fixed it, wait why do all tests fail with `only` not being allowed. :|

Reply:
For context, this is from the oclif generator. I think they are being a little aggressive with this and relying on the linting to deny `only` should be fine.